README.md

```diff
@@ -1,24 +1,185 @@
-# README
+# **ROR_CAPSTONE_BUDGET_APP** 
+With this App, one can manage budget: A list of transactions is there which associated with a category, so that one can see how much money to be spent for a particular choice.  A user is allowed to:  register and log in, so that the data is private to them. New category and new transaction can be created also.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# 📖 ROR_CAPSTONE_BUDGET_APP <a name="about-project"></a>
 
-* Database initialization
+ **ROR_CAPSTONE_BUDGET_APP** With this App, one can manage budget: A list of transactions is there which associated with a category, so that one can see how much money to be spent for a particular choice.  A user is allowed to:  register and log in, so that the data is private to them. New category and new transaction can be created also.
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">postgres</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Ruby on Rails
+- PostgreSQL Database
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> None
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221058794-389accdc-e110-436a-beb6-70b85beff8ae.png)


- [ ] It would be nice if you deploy your application because it was a requirement in your project details if you can see :+1: and add a link in this **Live Demo** section to make your readme file descriptive :+1: